formulas/formulas/australian.ts

```diff
@@ -44,14 +45,15 @@ export const australianFormula: Formula = {
     adjusters: [
         breakpointFCAdjuster
     ],
-    treatments: [
-        na_clo,
-        soda_ash,
-        baking_soda,
-        m_acid,
-        cal_chlor,
-        cya_treatment,
+    balanceOrder: [
+        DT('fc', na_clo, null),
+        DT('ph', soda_ash, m_acid),
+        DT('ta', baking_soda, m_acid),
+        DT('ch', cal_chlor, null),
+        DT('cya', cya_treatment, null),
+        DT('phosphate', phosphate_rem, null),
+    ],
```

Comment:
Instead of defining an ordered list of treatments, we provide an ordered list of readings to balance, along with default treatments for each direction of each.

This way, it's easy to swap things when substituting. If we want to just always check some treatments, that comes in the `alwaysCheck` property below. Those run after balancing.